pkg/sloop/webfiles/jdd.js

```diff
@@ -0,0 +1,438 @@
+(function($) {
```

Comment:
I don't see the usage of this library in this PR, can we remove it? Also when we start using it - can we reference it via html script tag?